### PR TITLE
Feat: Login, Signup, Forget Password

### PR DESCRIPTION
**Pull request by pritam-kr**
- Created login, signup, and forgot password page

- changed some styling on

- fixed routing

Screenshot (235)

Screenshot (236)

Screenshot (237)

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Good to go!!

File: commonstyle.css [hunk 1]
```diff
@@ -43,7 +43,7 @@ html {
   color: var(--text-color);
   justify-content: space-between;
   align-items: center;
-  padding: 2rem 5rem 1rem 5rem;
+  padding: 2rem 5rem 2rem 5rem;
```
Comment (srishti-maurya) [inline commonstyle.css:46]: you may right it as `padding: 2rem 5rem;`

File: commonstyle.css [hunk 2]
```diff
@@ -118,8 +120,8 @@ html {
   background-color: var(--wrong-ans);
 }
 
-.github-logo{
+.github-logo {
```
Comment (srishti-maurya) [inline commonstyle.css:123]: may be you can use `social-media-logo` in that it could be reused in future

File: Pages/forgot.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                <p class="paragraph"> <a href="/Pages/login.html" class="links">Back to SignIn</a> <i
+                        class="fas fa-chevron-right site-icon"></i></p>
+            </div>
+
+        </form>
+    </div>
+
+</body>
+
+</html>
```
Comment (varnita619) [inline Pages/forgot.html:50]: You can add new line at end of each file

**Review by varnita619 — APPROVED** [review]
Looks Good  to go

File: Pages/forgot.html (hunk 3 above)
Comment (pritam-kr) [reply Pages/forgot.html:50]: Ok

File: css/auth.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    background-color: var(--primary-color);
+}
+.btn-submit:hover{
+    background-color: var(--primary-color);
+}
+.links{
+    font-size: small;
+    color: var(--primary-color);
+}
+.links:hover{
```
Comment (srishti-maurya) [inline css/auth.css:29]: you may merge it with `btn-submit:hover` as it has the same property

File: css/auth.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+}
+.login-container{
+    min-height: 88vh;
+}
+
+.btn-submit{
+    background-color: var(--primary-color);
+}
+.btn-submit:hover{
+    background-color: var(--primary-color);
```
[the remark this answers is not in the document]
Comment (pritam-kr) [reply css/auth.css:23]: Yess!. I followed the style guide to override CSS. I don't use any hover effects.

**Review by srishti-maurya — APPROVED** [review]
looks good

File: css/auth.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (srishti-maurya) [reply css/auth.css:23]: ok got it! great 👍

**Comment by preethasri** [comment]
Looks good👍

**Pull request merged**